docs/knowledgebase/learn-substrate/off-chain-features.md

```diff
@@ -96,3 +97,10 @@ populating the storage every time a block is processed, so the data is always co
 exactly the same for every node with indexing enabled.
 
 ## Learn More
+
+To look at concrete examples of off-chain workers and how to use them in runtime development,
+refer to the following sections in Substrate Recipes:
+
+- <span id="example-txs">[Submit signed and unsigned transactions from off-chain workers back on-chain](https://substrate.dev/recipes/off-chain-workers/transactions.html)</span>
```

Comment:
These should be links to the documents in the local directory structure, Not to the full hosted URL.

Reply:
Ah unless it can't be, as recipes are hosted separately. So maybe not possible here.